openslides/assignments/static/js/assignments/site.js

```diff
@@ -527,20 +528,27 @@ angular.module('OpenSlidesApp.assignments.site', [
         };
         // create new ballot
         $scope.createBallot = function () {
-            $scope.activeTab = 0;
             $http.post('/rest/assignments/assignment/' + assignment.id + '/create_poll/').then(
                 function (success) {
                     $scope.alert.show = false;
                     if (assignment.phase === 0) {
                         $scope.updatePhase(1);
                     }
+                    $timeout(function() {
```

Comment:
Can you please add a short not with a link to this issue? Maybe they will fix it later (or maybe not if ui-bootstrap is a dead project (in favor of Angular 2).